data/sql/updates/db_world/2019_01_11_00.sql

```diff
@@ -0,0 +1,39 @@
+-- DB update 2019_01_10_00 -> 2019_01_11_00
+DROP PROCEDURE IF EXISTS `updateDb`;
+DELIMITER //
+CREATE PROCEDURE updateDb ()
+proc:BEGIN DECLARE OK VARCHAR(100) DEFAULT 'FALSE';
+SELECT COUNT(*) INTO @COLEXISTS
+FROM information_schema.COLUMNS
+WHERE TABLE_SCHEMA = DATABASE() AND TABLE_NAME = 'version_db_world' AND COLUMN_NAME = '2019_01_10_00';
+IF @COLEXISTS = 0 THEN LEAVE proc; END IF;
+START TRANSACTION;
+ALTER TABLE version_db_world CHANGE COLUMN 2019_01_10_00 2019_01_11_00 bit;
+SELECT sql_rev INTO OK FROM version_db_world WHERE sql_rev = '1547165890320901800'; IF OK <> 'FALSE' THEN LEAVE proc; END IF;
+--
+-- START UPDATING QUERIES
+--
+
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1547165890320901800');
+
+DELETE FROM `trinity_string` WHERE `entry` BETWEEN 5062 AND 5071;
+INSERT INTO `trinity_string` (`entry`, `content_default`) VALUES
+(5062, 'You will now read %s conversations.'),
+(5063, 'Stopped spying %s.')
```

Comment:
```sql
(5063, 'Stopped spying %s.'),
```

Reply:
Slipped right under my nose, thank you @Winfidonarleyan !